user/cinnamon-desktop/template.py

```diff
@@ -0,0 +1,41 @@
+pkgname = "cinnamon-desktop"
+pkgver = "6.2.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dpnp_ids=/usr/share/hwdata/pnp.ids"]
+hostmakedepends = ["gettext", "gobject-introspection", "meson", "pkgconf"]
+makedepends = [
+    "elogind-devel",
+    "gdk-pixbuf-devel",
+    "glib-devel",
+    "gtk+3-devel",
+    "libpulse-devel",
+    "libx11-devel",
+    "libxext-devel",
+    "libxkbfile-devel",
+    "libxrandr-devel",
+    "xkeyboard-config",
+]
+depends = ["chimera-artwork", "hwdata-pnp"]
+pkgdesc = "Cinnamon desktop library and common settings schemas"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later AND LGPL-2.0-or-later"
```

Comment:
some files (e.g. `libcinnamon-desktop/gnome-installer.c`) are under `LGPL-2.1-or-later`, so i'd do this
```suggestion
license = "GPL-2.0-or-later AND LGPL-2.1-or-later"
```